### ISSUE TITLE
vSphere Automation SDK not included, vmware tags not supported in inventory

### ISSUE THREAD

**Issue by anasskh** [ansible/awx#8734]
##### ISSUE TYPE
 - Bug Report

SUMMARY

VMware dynamic inventory pulling the tag with empty value 

STEPS TO REPRODUCE

Sync with vcenter 

##### ENVIRONMENT
Ansible 2.9.14
AWX Version 15.1 
vshpere 6.5 

Sources Variables: 

hostnames:
- config.name
keyed_groups:
- key: guest.guestFamily
  prefix: ''
  separator: ''
plugin: community.vmware.vmware_vm_inventory
properties:
- name
- network
- guest
- config
- tag
strict: false
with_nested_properties: true

EXPECTED RESULTS
I expected sync VM with tags value and filter with tag to create inventory by Environments.

ACTUAL RESULTS

i get tags with empty value like these tag: []

![image](https://user-images.githubusercontent.com/20398877/100923440-76c67100-34df-11eb-92ea-96cef2136c93.png)

Thanks for your time .

**Comment by wenottingham** [comment]
`vSphere Automation SDK` is required for tags, it is not included out of the box.

**Comment by anasskh** [comment]
yes it work when i install the vSphere Automation SDK direct on the container .

Thanks .
